src/v2/models/XcmModels.ts

```diff
@@ -23,6 +25,8 @@ export enum parachainIds {
   MOONBEAM = 2004,
   STATEMINE = 1000,
   STATEMINT = 1000,
+  KINTSUGI = 2092,
```

Comment:
Does it make sense to move parachain id to `src/modules/xcm/index.ts `where all parachains are defined?

Reply:
Yes, I wanted too, but now I'd like to avoid defining variables under `src/modules/xcm/index.ts` because importing variables from this file cause an error in unit tests.